roles/shrink_lv/tasks/check_device.yaml

```diff
@@ -1,20 +1,20 @@
 - name: Get the mount point info
   ansible.builtin.set_fact:
-    shrink_lv_mount_info: "{{ ansible_facts.mounts | selectattr('device', 'equalto', item.device) }}"
+    shrink_lv_mount_info: "{{ ansible_facts['mounts'] | selectattr('device', 'equalto', item['device']) | first }}"
```

Comment:
Is it guarantied that the mount we are looking for is the `first`? Is there a better way to make sure that we take the right one?

Reply:
OK, makes sense. Thanks